src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  errorType: string;
+  hasError: boolean;
+  setHasError: React.Dispatch<boolean>;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorType,
+  hasError,
+  setHasError,
+}) => {
+  const [isHidden, setIsHidden] = useState(true);
+  const emptyTitleNotification = 'Title can\'t be empty';
```

Comment:
useless variable actually, just move string where you use it

Reply:
fixed it